pkg/common/identifier_url.go

```diff
@@ -0,0 +1,20 @@
+package common
+
+import (
+	"fmt"
+	"strings"
+)
+
+// GetIdentifierURL returns the URL for a given identifier and value
+// Some Values need to be trimmed to use it in the URL
+func GetIdentifierURL(identifier, value string) string {
```

Comment:
This is "overloading" the purpose of this function. You should create separate functions for each. Example `GetCveReferenceURL()`, `GetCweReferenceURL` etc.

This suggestion is based on:
https://en.wikipedia.org/wiki/Single-responsibility_principle